src/main/resources/application.yaml

```diff
@@ -0,0 +1,125 @@
+server:
+    port: '8444'
+    servlet:
+        context-path: ${service.variable.contextPath}
+
+# Springboot Configuration
+spring:
+    application:
+        name: mcp-service-registry
+    jpa:
+        properties:
+            hibernate:
+                search:
+                    backend:
+                        lucene_version: LATEST
+                        directory:
+                            root: ./lucene/
+                        analysis:
+                            configurer: >-
+                                class:net.maritimeconnectivity.serviceregistry.config.MSRLuceneAnalysisConfigurer
+                    schema_management:
+                        strategy: validate
+        generate-ddl: true
+        hibernate:
+            ddl-auto: update
+            show-sql: true
+    datasource:
+        url: jdbc:${service.variable.datasource.server.type}://${service.variable.datasource.server.host}:${service.variable.datasource.server.port}/${service.variable.datasource.database.name}
+        username: ${service.variable.datasource.database.username}
+        password: ${service.variable.datasource.database.password}
+    flyway:
+        enabled: true
+        url: jdbc:${service.variable.datasource.server.type}://${service.variable.datasource.server.host}:${service.variable.datasource.server.port}/${service.variable.datasource.database.name}
+        schemas: mcp_service_registry
+        user: ${service.variable.datasource.database.username}
+        password: ${service.variable.datasource.database.password}
```

Comment:
```suggestion
```
Flyway will by default use the datasource that is defined above

Reply:
OK will remove that part